<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-image: url(&quot;C:/Users/isupo/Google Диск/Programming/Python Programming/Python Data Science/NeuroLand/ui/f3.jpg&quot;);
}</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>731</width>
      <height>32</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0.95);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
border-radius: 8px;
border: 1px solid rgb(217, 210, 0);
outline: none;
</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string> Введите ссылку на земельный участок</string>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>405</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'JetBrains Mono';&quot;&gt;1. Отсутствие общественного транспорта и удаленность от шоссе&lt;br/&gt;2. Непосредсвтенная близость к автомагистрали&lt;br/&gt;3. Наличие остановок общественного транспорта, транпорт ходит в интервале 1 часа&lt;br/&gt;4. Есть все виды транспорта, приезжающих в интервале 15 минут&lt;br/&gt;5. Наличие всех видов транспорта, приезжающих в интервале 5 и менее минут&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../../../.designer/backup/qq.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>474</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'JetBrains Mono';&quot;&gt;1. Загрязнённая местность, близость к заводам или к ЭС&lt;br/&gt;2. Наличие субботников, близоть к лесу&lt;br/&gt;3. Небольшое количество загрязнённых участков, наличие мусорных баков&lt;br/&gt;4. Территория не загрязнена, наличие дворников&lt;br/&gt;5. Чистая территория, удалённость от заводов и ЭС, большое количество деревьев и растений&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../../../.designer/backup/qq.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'JetBrains Mono';&quot;&gt;1. Отсутствие коммуникации&lt;br/&gt;2. Наличие холодной, горячей воды, газа, происходят частые перебои работы (нав ремя отключается горячая вода)&lt;br/&gt;3. Наличие холодной, горячей воды, газа, происходят частые перебои работы, но есть быстрый отклик работников на поломки&lt;br/&gt;4. Присутствует обычные услуги ЖКХ, по типу, уборка двора и его ремонт, наличие мусорных баков, мусоровоза и т.д. &lt;br/&gt;5. Присутствуют все необходимые удобства: происходят собрания жильцов домов, дворники, присутсвует быстрый отклик на поломки и т.д.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../../../.designer/backup/qq.png</pixmap>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>304</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="styleSheet">
        <string notr="true">font: 75 40pt &quot;MS Shell Dlg 2&quot;;
color: rgb(139, 30, 32);</string>
       </property>
       <property name="text">
        <string>N</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="styleSheet">
        <string notr="true">font: 75 40pt &quot;MS Shell Dlg 2&quot;;
/*color: rgb(90, 90, 90);</string>
       </property>
       <property name="text">
        <string>e</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_13">
       <property name="styleSheet">
        <string notr="true">font: 75 40pt &quot;MS Shell Dlg 2&quot;;
/*color: rgb(90, 90, 90);</string>
       </property>
       <property name="text">
        <string>u</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_14">
       <property name="styleSheet">
        <string notr="true">font: 75 40pt &quot;MS Shell Dlg 2&quot;;
/*color: rgb(90, 90, 90);</string>
       </property>
       <property name="text">
        <string>r</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_15">
       <property name="styleSheet">
        <string notr="true">font: 75 40pt &quot;MS Shell Dlg 2&quot;;
/*color: rgb(90, 90, 90);</string>
       </property>
       <property name="text">
        <string>o</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_16">
       <property name="styleSheet">
        <string notr="true">font: 75 40pt &quot;MS Shell Dlg 2&quot;;
color: rgb(139, 30, 32);</string>
       </property>
       <property name="text">
        <string>L</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_17">
       <property name="styleSheet">
        <string notr="true">font: 75 40pt &quot;MS Shell Dlg 2&quot;;
/*color: rgb(90, 90, 90);</string>
       </property>
       <property name="text">
        <string>a</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_18">
       <property name="styleSheet">
        <string notr="true">font: 75 40pt &quot;MS Shell Dlg 2&quot;;
/*color: rgb(90, 90, 90);</string>
       </property>
       <property name="text">
        <string>n</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_19">
       <property name="styleSheet">
        <string notr="true">font: 75 40pt &quot;MS Shell Dlg 2&quot;;
/*color: rgb(90, 90, 90);</string>
       </property>
       <property name="text">
        <string>d</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="find_button">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>280</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	/*background-color: orange;*/
	color: #fff;
	background: transparent;
	border: 2px solid #fff;
	border-radius: 20px;
	font: 20pt &quot;Stencil&quot;;
}
QPushButton:hover{
	border-color:#eee;
	color: #eee;
}
QPushButton:pressed{
	border-color: silver;
	color: silver;
}
</string>
    </property>
    <property name="text">
     <string>Find</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>551</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="cost1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 25pt &quot;Stencil&quot;;
margin-left: 4px;
padding-left: 0;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="result">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>431</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 100 44pt &quot;Stencil&quot;;
background-color: rgba(0, 0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="cost2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>271</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 24pt &quot;Stencil&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="website_cost">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>100</y>
       <width>241</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 28pt &quot;Stencil&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionAdvenced_Mode">
   <property name="text">
    <string>Advenced Mode</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resours.qrc"/>
 </resources>
 <connections/>
</ui>
